src/card/basic/card.html

```diff
@@ -1,15 +1,15 @@
 <div ng-class="$ctrl.showTopBorder === 'true' ? 'card-pf card-pf-accented' : 'card-pf'">
   <div ng-if="$ctrl.showHeader()" ng-class="$ctrl.shouldShowTitlesSeparator ? 'card-pf-heading' : 'card-pf-heading-no-bottom'">
-    <div ng-if="$ctrl.showFilterInHeader()" ng-include="'card/basic/card-filter.html'"></div>
+    <div ng-class="{'hide': $ctrl.showSpinner === true}" ng-if="$ctrl.showFilterInHeader()" ng-include="'card/basic/card-filter.html'"></div>
```

Comment:
This might be better written as:
`<div ng-if="$ctrl.showFilterInHeader() && !$ctrl.showSpinner" ...`
...or you could even move the logic into the $ctrl.showFilterInHeader() function.